app/src/main/java/com/mk/tasky/agenda/domain/model/AgendaItem.kt

```diff
@@ -0,0 +1,40 @@
+package com.mk.tasky.agenda.domain.model
+
+import java.time.LocalDateTime
+
+sealed class AgendaItem(
+    val id: String,
+    val title: String,
+    val description: String,
+    val firstDatetime: LocalDateTime,
```

Comment:
I find the name `firstDatetime` says quite little about what it represents. Would it be clear to you if you'd start working on this project if someone else wrote this?
But I also struggled to find a good name for this, which is why I came up with `sortDate`, pretty much the date we want to sort the AgendaItems by on the AgendaScreen. Not ideal either, but what about just calling it `time` or `dateTime` and then Events just have another field called `toTime`? 🤔

Reply:
Renamed to time